docs/UserGuide.md

```diff
@@ -160,6 +122,89 @@ Expected output (fail):
 ```agsl
 Oops! Invalid search expression, please check again.
 ```
+### Viewing a fosterer's detail : `view`
+
+Views a fosterer's details by shifting the foster family list view page to a fosterer's profile view page. 
+
+Format: `view INDEX` 
+
+Parameters:
+* `INDEX`: Index of a fosterer to view their details shown in the list obtained by the find/list command
+
+<div markdown="span" class="alert alert-primary">
+  :exclamation: <b>Important:</b>
+Some commands are not available while on profile view page. 
+The list of available commands are: 
+<ul>
+  <li> <code>help</code> </li>
+  <li> <code>edit</code> </li>
+  <li> <code>exit</code> command to <b>exit the profile view page</b> </li>
+</ul>
+</div> 
+<br>
+
+Example: 
+* `list` followed by `view 2` to view the profile of the 2nd fosterer in the address book. 
+
+Expected output (success):
+```agsl
+Viewing the profile of a fosterer...
+```
+
+Expected output (fail):
+```agsl
+Oops! Invalid fosterer index provided, please check again.
+```
+
+### Editing a fosterer's detail : `edit`
+
+Edits or views the details of a fosterer stored in the address book. 
+
+Format 1: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [animal/ANIMAL_NAME] [t/AVAILABILITY] [t/HOUSING_TYPE] [t/TYPE_OF_ANIMAL_FOSTERED/WANTED…]`
+
+Format 2: `edit INDEX` 
+
+Parameters: 
+* `INDEX`: Index of a fosterer to be edited shown in the list obtained by the find/list command
+* `NAME`: New name of the fosterer
+* `PHONE_NUMBER`: New phone number of the fosterer
+* `EMAIL`: New email of the fosterer
+* `ADDRESS`: New address of the foster family
+* `ANIMAL_NAME`: New name of the animal fostered (if fosterer is currently fostering an animal / “NotAvailable”)
+* `AVAILABILITY`: Available / NotAvailable
+* `HOUSING_TYPE`: HDB / Condo / Landed
+* `TYPE_OF_ANIMAL_FOSTERED` (if not available): current.Dog / current.Cat
+* `TYPE_OF_ANIMAL_WANTED` (if available): able.Dog / able.Cat
+
+<div markdown="span" class="alert alert-primary">
+  :bulb: <b>Tip:</b>
+The index of the fosterer has to be provided, however the number of parameters to be edited can be variable from none to changing every field.
```

Comment:
maybe a clearer way to phrase this line can be: "...but the number of parameters to be edited can vary from zero to all fields." what do you think!